### ISSUE TITLE
logging: project quota limits caused flaky TestLogsAndDelete

### ISSUE THREAD

**Issue by odeke-em** [googleapis/google-cloud-go#1589]
I just noticed this in https://source.cloud.google.com/results/invocations/d1a9e995-2251-4c81-b0c5-20c5b6b87a5d/targets/tests/tests;group=logging;test=TestLogsAndDelete;row=1 originally from https://source.cloud.google.com/results/invocations/60e6edc6-d114-4400-9020-62236f37ab9e/targets/cloud-go-libraries%2Fgoogle-cloud-go%2Fcontinuous%2Fmaster/log
```shell
logging_test.go:497: deleting "GO-LOGGING-CLIENT/TEST-LOG-20190919-73553409892752-0001": 
rpc error: code = ResourceExhausted desc = Quota exceeded for quota metric 
'logging.googleapis.com/control_requests' and limit 'ControlRequestsPerMinutePerProject' of 
service 'logging.googleapis.com' for consumer 'project_number:36639933145'.
```

@jadekler @broady could we perhaps take a look at the quota limits and perhaps bump them up?

That Kokoro run identified a whole bunch of other flakes
<img width="316" alt="Screen Shot 2019-09-19 at 7 58 19 PM" src="https://user-images.githubusercontent.com/4898263/65295951-da04f700-db17-11e9-8da6-ee5c326bc3e3.png">

**Comment by broady** [comment]
Looks like the quota is already maxed out:
![image](https://user-images.githubusercontent.com/24982/65389338-008a8400-dd0a-11e9-9b14-caa05a09a953.png)

I think we can request a quota bump separately.
Should also consider using a project pool (see gimmeproj in golang-samples)

**Comment by codyoss** [comment]
Will be fixed under #1654
